http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonBPayloadIT.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.services.QuarkusApplication;
+
+public class JsonBPayloadIT extends JsonPayloadIT {
+
+    @QuarkusApplication(classes = { FootballTeamResource.class, FootballTeam.class,
+            Person.class,
```

Comment:
Afaik I meant just to move `PersonResource.class` to new line and leave`Person.class,` as it was.
So the result should be looking like 

```
@QuarkusApplication(classes = { FootballTeamResource.class, FootballTeam.class, Person.class, 
            PersonResource.class }, properties = "oidcdisable.properties")
 ```
 
 If you want you can change it like this

 ```
@QuarkusApplication(classes = { FootballTeamResource.class, FootballTeam.class, 
            Person.class, PersonResource.class }, properties = "oidcdisable.properties")
 ```